backend/internal/core/domain/gems/gem.go

```diff
@@ -4,6 +4,17 @@ import (
 	"time"
 )
 
+type GemColor int64
+
+const (
+	Purple GemColor = 0
```

Comment:
As a side note, `iota` is a possible alternative, though I think it makes more sense to list out each value given that they need to map to the same values in the frontend.